app/models/event.rb

```diff
@@ -240,5 +241,11 @@ def user_cannot_be_inactive
 
     errors.add(:base, "You need an active subscription to capture events with your live API key")
   end
+
+  def property_cannot_be_all
```

Comment:
Let's call the method someting general, so we can add more values in the future if needed. E.g. `property_name_cannot_be_excluded`.